cps/src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition2Test.java

```diff
@@ -46,8 +47,9 @@
 
         assertFalse("Expected the execution to be suspended but it has completed", exec.isComplete());
 
+        FlowExecutionOwner owner = exec.getOwner();
         exec = roundtripXStream(exec);    // poor man's simulation of Jenkins restart
```

Comment:
`RestartableJenkinsRule`, yes. I doubt that actually applies in this test, however, since the objects are all mocks not bound to Jenkins disk storage.